pulsar-broker/src/test/java/org/apache/pulsar/broker/resourcegroup/ResourceUsageTransportManagerTest.java

```diff
@@ -113,8 +113,8 @@ public void acceptResourceUsage(String broker, ResourceUsage resourceUsage) {
     }
 
     private void prepareData() throws PulsarAdminException {
-        this.conf.setResourceUsageTransportClassName("org.apache.pulsar.broker.resourcegroup.ResourceUsageTransportManager");
+        this.conf.setResourceUsageTransportClassName("org.apache.pulsar.broker.resourcegroup.ResourceUsageTopicTransportManager");
         this.conf.setResourceUsageTransportPublishIntervalInSecs(PUBLISH_INTERVAL_SECS);
         admin.clusters().createCluster("test", new ClusterData(pulsar.getBrokerServiceUrl()));
     }
-}
\ No newline at end of file
+}
```

Comment:
Overall: org.apache.pulsar.broker.resourcegroup.ResourceGroupConfigInfo class (the entire file) can now be deleted?

Reply:
yes, I plan to do it in my next commit.